.github/checkgroup.yml

```diff
@@ -0,0 +1,134 @@
+custom_service_name: "Lightning CI required checker"
+subprojects:
+  - id: "pytorch_lightning"
```

Comment:
Shall we also separate docs for each as changes of docs do not need to run full code testing...

Reply:
Correct, let's do that